src/eos-dropbox-app

```diff
@@ -186,11 +192,13 @@ class DropboxLauncher():
                                            'org.freedesktop.portal.OpenURI',
                                            None)
 
-            logging.info("Opening Dropbox directory at {}...".format(uri))
-            handle = proxy.OpenURI('(ssa{sv})',
-                                   '',                          # Parent window ID
-                                   uri,                         # URI
-                                   GLib.Variant('a{sv}', None)) # Options
+            logging.info("Opening Dropbox directory at {}...".format(path))
+            handle = proxy.call_with_unix_fd_list('OpenFile',
```

Comment:
Technically we should bump the `Depends` in `debian/control` to `xdg-desktop-portal >= 0.6` or something like that.

But given that the dependency isn’t already declared, and I’m currently reviewing an appropriate upgrade of the xdg-desktop-portal, things aren’t getting any worse with this PR.

Reply:
Good point, I'll add a change to bump this

Reply:
Actually I think I'll leave it as it is now, because this is meant to be bundled inside the dropbox flatpak and it's not that putting a dependency on xdg-desktop-portal in the debian/control will make a difference...